content/en/apps/reference/app-settings/_index.md

```diff
@@ -28,19 +28,20 @@ The following settings do not need to be specified. They should only be defined
+| outgoing_deny_with_alphas        | When `true`, deny all messages to recipient phone numbers containing letters (eg. `Safaricom`). Intended to avoid [message loops]({{% ref "apps/guides/messaging/message-loops" %}}) with non-numeric senders used by gateways. A boolean.                                                                                                                                                                                              | true | 3.3.0   |
+| outgoing_deny_with_alphas        | When `true`, deny all messages to recipient phone numbers containing letters (eg. `Safaricom`). Intended to avoid [message loops]({{% ref "apps/guides/messaging/message-loops" %}}) with non-numeric senders used by gateways. A boolean.                                                                                                                                                                                              | true | 3.3.0   |
+| task_day_limit                   | The number of days before a task is due to show the due date.                                                                                                                                                                                                                                                                                                                                                                           | 4 | 3.9.0   |
+| app_url                          | The URL of the app, eg: "https://demo.app.medicmobile.org"                                                                                                                                                                                                                                                                                                                                                                              | | 3.10.0  |
+| task_days_overdue                | Display number of overdue days in tasks list                                                                                                                                                                                                                                                                                                                                                                                            | false | 3.13.0  |
+| languages                        | Array of objects with `locale` and `enabled` properties representing respectively the 2 or 3 letter language code and whether that language should be enabled.  <br/>If unset it falls back to the previous behavior of relying on the `enabled` property of each translation document `messages-XX.properties`. This fallback behavior will be deprecated and this property will be required starting with the next major version 5.0. | | 4.2.0   |
```

Comment:
```suggestion
| languages                        | Array of objects with `locale` and `enabled` properties representing respectively the 2 or 3 letter language code and whether that language should be enabled.  <br/>If unset it falls back to the previous behavior of relying on the `enabled` property of each translation document `messages-XX.properties`. This fallback behavior is now deprecated and will be removed in the next major version (5.0). This `languages` configuration property will be required for CHT 5.0+. | | 4.2.0   |
```

Minor, but I think the `enabled` in the translation documents is considered "deprecated" _now_ (since it has been superseded by a new go-forward config).  Then, in `5.0` it will be "removed".